components/camel-sjms/src/test/java/org/apache/camel/component/sjms/support/SjmsConnectionTestSupport.java

```diff
@@ -55,8 +55,6 @@ public void setup() throws Exception {
                 || getConnectionUri().startsWith("vm")) {
             vmTestConnectionFactory = new ActiveMQConnectionFactory(
                     VM_BROKER_CONNECT_STRING);
-        } else {
-            createBroker();
```

Comment:
Is this correct ?

Reply:
@davsclaus in theory no, but I think this class is unused. 

I think the whole class can be removed. It does not seem to be used anywhere else. Please, can you give it a try @dk2k ?

Reply:
Yes. If I remove unused assignment in createBroker(), then I can remove all it's code. Eventually I can delete the method and its call

Reply:
@dk2k I triggered a new build, but I don't think it's related to your changes.